wled00/data/settings_wifi.htm

```diff
@@ -6,18 +6,100 @@
 	<meta content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no" name="viewport"/>
 	<title>WiFi Settings</title>
 	<script>
-		var d=document;
-		var loc = false, locip;
+		let d = document;
+		let loc = false, locip;
+		let scanLoops = 0;
+
+		function Local() {
+			if (window.location.protocol == "file:") {
+				loc = true;
+				locip = localStorage.getItem("locIp");
+				if (!locip) {
+					locip = prompt("File Mode. Please enter WLED IP!");
+					localStorage.setItem("locIp", locip);
+				}
+			}
+		}
 		function H(){window.open("https://kno.wled.ge/features/settings/#wifi-settings");}
 		function B(){window.open("/settings","_self");}
+		function N() {
+			const url = (loc?`http://${locip}`:"") + "/json/networks";
```

Comment:
If it must be `/json` then just shorten it to `/json/net`. I would prefer it elsewhere if possible, though.
See comments below.

Reply:
I think putting it within the `/json` url prefix makes the most sense from an api point of view, but with memory taken into account, would moving it to `/net` make the more sense and dispatching to its own handler?

For now I'll just make it the shorter `/json/net`, but open to moving it all together.